flink-runtime/src/test/java/org/apache/flink/runtime/deployment/ResultPartitionDeploymentDescriptorTest.java

```diff
@@ -40,26 +50,47 @@
 	public void testSerialization() throws Exception {
 		// Expected values
 		IntermediateDataSetID resultId = new IntermediateDataSetID();
+
 		IntermediateResultPartitionID partitionId = new IntermediateResultPartitionID();
+		ExecutionAttemptID producerExecutionId = new ExecutionAttemptID();
+
 		ResultPartitionType partitionType = ResultPartitionType.PIPELINED;
 		int numberOfSubpartitions = 24;
+		int connectionIndex = 10;
+
+		PartitionDescriptor partitionDescriptor = new PartitionDescriptor(
+			resultId,
+			partitionId,
+			partitionType,
+			numberOfSubpartitions,
+			connectionIndex);
+
+		ResourceID producerLocation = new ResourceID("producerLocation");
+		InetSocketAddress address = new InetSocketAddress("localhost", 10000);
+		ResultPartitionID resultPartitionID = new ResultPartitionID(partitionId, producerExecutionId);
+		ConnectionID connectionID = new ConnectionID(address, connectionIndex);
+		ShuffleDescriptor shuffleDescriptor = new NettyShuffleDescriptor(
```

Comment:
Not sure we should also cover the case of `UnknownShuffleDescriptor` in another test